frontend/src/components/Movie.jsx

```diff
@@ -1,23 +1,28 @@
 import React from "react";
 import PropTypes from "prop-types";
-
+// fonction d'affichage du titre, image et description du film
 function Movie({ movie }) {
   return (
-    <section>
-      <h1>{movie.title}</h1>
-      <img
-        src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
-        alt={movie.path}
-      />
-    </section>
+    <div>
+      <h5>{movie.title}</h5>
+      <div className="container">
+        <img
+          src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
+          alt={movie.path}
+          className="imagetohover"
+        />
+        <div className="middle">
+          <div className="texthover">{movie.overview}</div>
+        </div>
+      </div>
```

Comment:
a container for an image and related data : it's a figure, not a div

Reply:
done

Reply:
done